modules/rosa-view-cloudwatch-logs.adoc

```diff
@@ -0,0 +1,22 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// rosa_logging/rosa-install-logging.adoc
+
+[id="rosa-view-cloudwatch-logs_{context}"]
+= Viewing forwarded logs
+
+.Prerequisites
+
+* The `cluster-logging-operator` add-on service is installed.
+* The Amazon CloudWatch add-on service is installed.
+
+.Procedure
+
+To view logs that have been forwarded to Amazon Web Services (AWS) from {product-title}:
+
+. Log in to the AWS console.
+. Select the region the cluster is deployed in.
+. Select the `CloudWatch` service.
+. Select `Logs` from the left column, and click `Log Groups`.
+. Select a log group explore. You can view application, infrastructure, or audit logs, depending on which types were enabled during the add-on service installation See the link:https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/WhatIsCloudWatch.html[Amazon CloudWatch User Guide] for more information.
```

Comment:
Just checking that "log group explore" is correct? Should it be "to explore"?

Reply:
Yup. Fixed.